include/eve/module/elliptic/ellint_fe.hpp

```diff
@@ -0,0 +1,166 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/arch.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
+#include <eve/module/core.hpp>
+#include <eve/module/elliptic/ellint_rf.hpp>
+#include <eve/module/math.hpp>
+#include <eve/traits/common_value.hpp>
+
+namespace eve
+{
+  template<typename Options>
+  struct ellint_fe_t : elementwise_callable<ellint_fe_t, Options, modular_option, eccentric_option, threshold_option>
+  {
+    template<eve::floating_value T>
+    constexpr EVE_FORCEINLINE
+    auto operator()(T a) const noexcept -> decltype(eve::zip(T(), T()))
+    { return EVE_DISPATCH_CALL(a); }
+
+    template<eve::floating_value T0, eve::floating_value T1>
+    requires (same_lanes_or_scalar<T0, T1>)
+    constexpr EVE_FORCEINLINE
+    auto operator()(T0 a, T1 b) const noexcept// -> decltype(kumi::zip<eve::common_value_t<T0, T1>, eve::common_value_t<T0, T1>>())
```

Comment:
this return type needs to be properly specified. I think we do somethign similar elsewhere

Reply:
done